src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -285,19 +280,28 @@ private GapicSurfaceTestClassView createUnitTestClassView(SurfaceTransformerCont
     for (Method method : context.getSupportedMethods()) {
       MethodConfig methodConfig = context.getMethodConfig(method);
       if (MethodConfig.isGrpcStreamingMethod(method)) {
+        addGrpcStreamingTestImport(context);
         MethodTransformerContext methodContext = context.asRequestMethodContext(method);
         testCaseViews.add(
-            createTestCaseView(
-                methodContext, testNameTable, methodConfig.getRequiredFieldConfigs()));
+            mockServiceTransformer.createTestCaseView(
+                methodContext,
+                testNameTable,
+                methodConfig.getRequiredFieldConfigs(),
+                ApiMethodType.CallableMethod,
+                InitCodeOutputType.SingleObject));
       } else if (methodConfig.isFlattening()) {
         for (FlatteningConfig flatteningGroup : methodConfig.getFlatteningConfigs()) {
           MethodTransformerContext methodContext =
               context.asFlattenedMethodContext(method, flatteningGroup);
           testCaseViews.add(
-              createTestCaseView(
+              mockServiceTransformer.createTestCaseView(
                   methodContext,
                   testNameTable,
-                  flatteningGroup.getFlattenedFieldConfigs().values()));
+                  flatteningGroup.getFlattenedFieldConfigs().values(),
+                  methodConfig.isPageStreaming()
+                      ? ApiMethodType.PagedFlattenedMethod
+                      : ApiMethodType.FlattenedMethod,
```

Comment:
Create a local var for the method type - the embedded structure is harder to follow. Also, FYI, I am renaming `ApiMethodType` to `ClientMethodType` in my PR https://github.com/googleapis/toolkit/pull/755 .